react/fields/Checkbox/Checkbox.less

```diff
@@ -0,0 +1,47 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  display: inline-block;
+}
+
+.input {
+  opacity: 0;
+  position: absolute;
+}
+
+.label {
+  .touchableText(@standard-type-scale);
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+}
+
+.checkBox {
+  width: @grid-row-height * 3;
+  height: @grid-row-height * 3;
+  border: @field-border-width solid @sk-mid-gray-light;
+  border-radius: @field-border-radius;
+  margin-right: @field-gutter-width;
+  padding: @field-border-radius;
+  fill: @sk-pink;
+
+
+  .input:not([readonly]):not([disabled]):focus + .label &, .rootFocus & {
+    outline: none;
+    border-color: transparent;
+    .focus();
+  }
+}
+
+
+.checkMark {
+  display: block;
+  width: initial;
+  height: initial;
+  fill: @sk-pink;
+  opacity: 0;
+}
+
+.checked {
+  opacity: 1;
```

Comment:
Instead of adding an extra class here you use the `:checked` selector. 
https://css-tricks.com/almanac/selectors/c/checked/